ui/src/app/components/runs/filters/number-range-filter/number-range-filter.component.html

```diff
@@ -0,0 +1,28 @@
+<!--
```

Comment:
I think the filters should not be located under `runs`, but directly under `components`

Reply:
As discussed we will extract interface of it in next "sprint"